package/base-files/files/etc/profile

```diff
@@ -49,11 +49,18 @@ in order to prevent unauthorized SSH logins.
 EOF
 fi
 
-service() {
-	[ -f "/etc/init.d/$1" ] || {
+service() { 
+	if [ "$1" = "enabled" ] && [ -z "$2" ]; then 
```

Comment:
Seems confusing to me, this is going to be called as `service enabled` and list enabled AND disabled services. 

Perhaps something like `service show-enabled` (or `list-enabled`) and `service show-disabled` (or `list-disabled`) would make more sense?

Reply:
Oh, great idea, perhaps list-enabled to be consistent with opkg's list-installed?

Unless there are other comments, I'll implement the fix and push it later this week.

Reply:
Implemented list-enabled and list-disabled as per our conversation.